mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -509,4 +523,45 @@ public void setImage(String fullName) {
         mIvUserImage.setImageDrawable(drawable);
         hideProgressDialog();
     }
+    public void savebitmap(Bitmap bitmapImage) {
```

Comment:
* Method doesn't need to be `public`.

* Do File I/O on a separate thread not on the main thread.

Reply:
Created ImageInputOutput.java class  👍